tests/utils.py

```diff
@@ -63,6 +64,29 @@ def get_json(response):
     return json.loads(response.get_data(as_text=True))
 
 
+def post_dict_to(client, endpoint, input_dict, headers=None):
```

Comment:
as in most tests we used the same header (`json_header`) this can be the default for the headers parameters

Reply:
the function name seems a bit cryptic to me, I would also change input_dict as request_body, data_body or similar.

Reply:
`post_dict_to()` name could be:
* post_request() / post_request_to()
* posting() / posting_to()
* post() / post_to()
* postdata() / post_data() / post_data_to()

I prefer "postdata".

Ok for `json_header`. What about just `data` instead of `input_dict` ?

Reply:
perfekt